Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM python:3.7-alpine
-ENV PACKAGES "bash curl openssl ca-certificates jq vim"
-RUN apk add --update "${PACKAGES}" && rm -rf /var/cache/apk/*
```

Comment:
We'll still want to clean up the apk cache after installation to keep the image small. Please can you add the `rm -rf /var/cache/apk/*` back in again. Thanks.